gpu_wrapper/chpl_cublas/cuBLAS.chpl

```diff
@@ -0,0 +1,481 @@
+module cuBLAS {
+
+  use C_CUBLAS;
+  public use SysCTypes;
+
+  proc cublas_create_handle(){
+    return cublas_create();
+  }
+
+  proc cublas_destroy_handle(handle: c_void_ptr){
+    cublas_destroy(handle);
+  }
+
+  proc cu_array(size: size_t){
+     require "c_cublas.h", "c_cublas.o";
```

Comment:
Might be worth sanity checking that we can compile a program that uses the cublas module but doesn't call any functions.